Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 93 style the projects page

## Related ticket:
[SCRUM-93](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-93?atlOrigin=eyJpIjoiNzQ5YjhhNTVjNjNmNGJlZThjMmY4NjRkNDQ2NTFlMWIiLCJwIjoiaiJ9)

## In this PR:
- Data fetching for Projects page
- Create Project cards, Project details page components
- Display customer logos in Our customers section

## Testing instruction
- Checkout branch
- Import latest db and config
- Check the Projects service page to see if everything still working fine.

[SCRUM-93]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-93?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
